### PR TITLE
Assessment Improvement

### PR DESCRIPTION
**Pull request by reesmanp**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+.idea/
```
Comment (jodth07) [inline .gitignore:1]: can we add a .gitignore too account for local values?

**Review by jodth07 — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (reesmanp) [reply .gitignore:1]: what do you mean?

Comment (jodth07) [reply .gitignore:1]: .gitignore to account for the .idea and local files that don't need to be pushed to the repos.

**Review by jodth07 — APPROVED** [review]

File: .gitignore (hunk 1 above)
Comment (reesmanp) [reply .gitignore:1]: this IS the .gitignore

**Pull request merged**